forecasts.csv

```diff
@@ -0,0 +1,193 @@
+power_wh
```

Comment:
It would be best not upload the output from the forecasts. Also their appears to be only the "power_wh" column?